src/JssBlazor.Components/Extensions/ComponentsApplicationBuilderExtensions.cs

```diff
@@ -1,22 +1,18 @@
-using Microsoft.AspNetCore.Components;
-using Microsoft.AspNetCore.Components.Builder;
+using Microsoft.AspNetCore.Blazor.Hosting;
 using Toolbelt.Blazor.Extensions.DependencyInjection;
 
 namespace JssBlazor.Components.Extensions
 {
     public static class ComponentsApplicationBuilderExtensions
     {
-        public static void UseJssBlazorComponents<TComponent>(
-            this IComponentsApplicationBuilder app,
-            string domElementSelector)
-            where TComponent : IComponent
```

Comment:
The DOM element selector shouldn't go away--users should be able to specify their own DOM element selector for their app. Has this functionality been removed from Blazor?

Reply:
It hasn't been removed, just moved to the WebAssemblyHostBuilder.  I have separated the functionality into 2 methods (including the version with DOM element selector) and renamed the file to something a bit more generic.  Let me know if you prefer different naming for the methods/file.